pages/services/beta-dcos-monitoring/0.4.3-beta/operations/grafana/storage/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: layout.pug
+navigationTitle: Grafana Storage
+title: Grafana Storage
+menuWeight: 20
+excerpt: Grafana Storage
+render: mustache
+model: ../../../data.yml
+---
+
+#include /services/include/beta-software-warning.tmpl
+
+# Configuring Grafana Storage
+
+## DSS Volume
+
+You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+
+At least one volume must be available for Grafana to use. For more on creating volumes, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-volume/dcos-storage-volume-create).
```

Comment:
Is there any minimal requirements for this volume that we should recommend? Should Grafana volumes be on high performance storage (e.g. SSD backed)? Is the example size below (512MB) sufficient? Or should it be larger. I would definitely include a recommendation for storage guidelines. I would also align the example to your guidelines.